README.md

```diff
@@ -23,7 +23,7 @@ differently with an easy to use yet very powerful syntax.
 
 ## Usage
 
-Create an instance based on an existing `RestTemplate`:
+Create an instance based on an existing `RestTemplate` (also see section *RestTemplate Configuration*):
```

Comment:
Thinking about it, I'd like to see the setup code with the `PassThroughResponseErrorHandler` up here. The OAuth2 stuff could be its own section below (and linked from here).